test/test-data/unit/graphene_plugin.test

```diff
@@ -1349,3 +1349,30 @@ class Employee(ObjectType[PersonModel]):
 [out]
 Success: no issues found in 1 source file 
 
+
+[case test_default_resolver_check_does_not_run_on_interface_fields]
+from typing import Optional, List as ListType, Type
+
+from graphene import Interface, Field, ResolveInfo, Argument, Boolean, Int, String, ObjectType
+
+
+class PersonModel:
+    name: str
+
+
+class Person(Interface[PersonModel]):
+    name = Field(String, required=True)
+    age = Field(Int, required=True)
+
+    @staticmethod
+    def resolve_age(_: PersonModel, __: ResolveInfo) -> int:
+        return 28 
+
+
+class Employee(ObjectType[PersonModel]):
+    class Meta:
+        interfaces = (Person,)
+
+
+[out]
+Success: no issues found in 1 source file 
```

Comment:
Is it allowed to override a resolver's behavior in an implementing class?

I.e. in Employee is it legal to make a `resolve_age` function that explicitly overrides the one on the interface Person?

If so, would it be valuable to have a test covering that the following is invalid:
```
from typing import Optional, List as ListType, Type

from graphene import Interface, Field, ResolveInfo, Argument, Boolean, Int, String, ObjectType


class PersonModel:
    name: str

class Person(Interface[PersonModel]):
    name = Field(String, required=True)
    age = Field(Int, required=True)

    @staticmethod
    def resolve_age(_: PersonModel, __: ResolveInfo) -> int:
        return 28 


class Employee(ObjectType[PersonModel]):
    class Meta:
        interfaces = (Person,)
    
    @staticmethod
    def resolve_age(_: PersonModel, __: ResolveInfo) -> str:
        return '35'

```

Reply:
More of a curiosity question - what happens if in the Employee ObjectType, we define a Field called age that is of type Field(String), directly contradicting the interface?  Does graphene throw at runtime?

Reply:
These are good questions! I think the first thing you mentioned (overriding a resolver) _is_ legal. As for the second, I'm not sure, but I would guess not.

There are a whole slew of interface-related checks I want to add soon. That would be a good one too!